bin/reth/src/node/events.rs

```diff
@@ -138,18 +138,22 @@ impl NodeState {
     }
 
     fn handle_consensus_layer_health_event(&self, event: ConsensusLayerHealthEvent) {
-        match event {
-            ConsensusLayerHealthEvent::NeverSeen => {
-                warn!("Post-merge network, but never seen beacon client. Please launch one to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::HasNotBeenSeenForAWhile(period) => {
-                warn!(?period, "Post-merge network, but no beacon client seen for a while. Please launch one to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::NeverReceivedUpdates => {
-                warn!("Beacon client online, but never received consensus updates. Please ensure your beacon client is operational to follow the chain!")
-            }
-            ConsensusLayerHealthEvent::HaveNotReceivedUpdatesForAWhile(period) => {
-                warn!(?period, "Beacon client online, but no consensus updates received for a while. Please fix your beacon client to follow the chain!")
+        // If pipeline is running, no consensus layer messages are expected to be received.
```

Comment:
sorry to bikeshed here, but I think this is still not accurate because we know that some clients still send us messages.

this comment should reflect that it _can_ be the case not that it is expected